pages/4_AWS_Health.py

```
@@ -1,34 +1,59 @@
 import pandas as pd
+import io
 
-st.title("AWS Health :material/health_and_safety:")
+# TODO: i) Saving File(s) to parquet/ORC [Done].
+#       ii) Sending SES Alerts to interested parties.
+#       iii) Adding option to send the report to S3.
+#       iv) Add functionality of seeing all used services in the account.
 
-events = get_all_events()
-df = pd.DataFrame(events)
+st.title("AWS Health :material/health_and_safety:")
```

Ruff (0.8.2)

`st` may be undefined, or defined from star imports

(F405)